modules/android/vsandroid_vcxproj.lua

```diff
@@ -211,19 +236,38 @@
 			elements = table.join(elements, {
 				android.debugInformation,
 				android.strictAliasing,
-				android.thumbMode,
 				android.fpu,
 				android.pic,
---				android.ShortEnums,
+				android.shortEnums,
+				android.cStandard,
+				android.cppStandard,
 			})
 			if _ACTION >= "vs2015" then
 				table.remove(elements, table.indexof(elements, vc2010.debugInformationFormat))
-				table.remove(elements, table.indexof(elements, android.thumbMode))
+
+				elements = table.difference(elements, {
+					-- Android has C[pp]LanguageStandard instead.
+					vc2010.languageStandard,
+					-- Ignore multiProcessorCompilation for android projects, they use UseMultiToolTask instead.
+					vc2010.multiProcessorCompilation,
+					-- minimalRebuild also ends up in android projects somehow.
+					vc2010.minimalRebuild,
+				})
+
+				-- VS has NEON support through EnableNeonCodegen.
+				table.replace(elements, vc2010.enableEnhancedInstructionSet, android.enableEnhancedInstructionSet)
+				-- precompiledHeaderFile support.
+				table.replace(elements, vc2010.precompiledHeaderFile, android.precompiledHeaderFile)
```

Comment:
This isn't wrong, but it feels inconsistent with how this and other modules work, which is to override the existing functions and change or extend their behaviour - there's plenty of examples of this in this file, including this function.

Reply:
Should I switch over to the class `premake.override` calls for this? I went with `table.replace` as it seemed more convenient to me and I've seen some modules (not the official ones) do it like this.

Reply:
Hmm, looks like GitHub ate my reply. This is probably fine, I prefer `premake.override` because I'm used to it but there isn't really much of a difference between the two approaches. I've marked this as resolved.